ext/XS-APItest/APItest.xs

```diff
@@ -1,3 +1,7 @@
+/*
+ * ex: set ts=8 sts=4 sw=4 et:
+ */
+
```

Comment:
Does this belong in the root `.editorconfig` instead?

Reply:
Yeah no harm in adding it there too. We already set it for .c and .h files but apparently not .xs.